tests/test_baker.py

```diff
@@ -393,17 +378,14 @@ def test_chaining_bulk_create_reduces_query_count(self):
                 _quantity=5,
                 _bulk_create=True,
             )
-            assert queries.count == 3  # 2 bulk create and 1 select on Person
+            # 2 bulk create and 1 select on Person
```

Comment:
I would move this comment to the beginning of the assert num queries block so it is more clear.

Reply:
Hi @timjklein36, should I create another PR for this change? It is not allowing me to add it as a commit here.

Reply:
@Edald123 Ah, I see I was late to the party and it was already merged. Don't worry about it. It is very minor and should still be relatively clear to anyone reading the test.